packages/fxa-support-panel/config/index.ts

```diff
@@ -68,7 +66,6 @@ const conf = convict({
       format: 'port',
     },
     publicUrl: {
-      default: 'http://localhost:3031',
```

Comment:
Not sure where these went. (Do we run support panel locally?)

Reply:
support panel runs on port `7100`. I removed this cus it's unused, guessing it was copied over from payment-server config since `3031` is from over there